kratos/spaces/ublas_space.h

```diff
@@ -249,23 +250,71 @@ class UblasSpace
 
     static TDataType TwoNorm(VectorType const& rX)
     {
-        return sqrt(Dot(rX, rX));
+        return std::sqrt(Dot(rX, rX));
     }
 
     static TDataType TwoNorm(MatrixType const& rA) // Frobenious norm
     {
         TDataType aux_sum = TDataType();
-
-        for (unsigned int i = 1; i < rA.size1(); i++)
+#ifndef _OPENMP
+        for (std::size_t i = 0; i < rA.size1(); i++)
         {
-            for (unsigned int j = 1; j < rA.size2(); j++)
+            for (std::size_t j = 0; j < rA.size2(); j++)
             {
                 aux_sum += rA(i,j) * rA(i,j);
             }
         }
-
+#else
+        #pragma omp parallel for
```

Comment:
Two comments here

1 - use int in the loop as otherwise it will not compile in win
2 - use an omp reduction instead of a sum

Reply:
std::size_t it compiles according to @roigcarlo